client/landing/jetpack-cloud/sections/backups/utils.js

```diff
@@ -102,7 +102,7 @@ export const getMetaDiffForDailyBackup = ( logs, date ) => {
 
 	return [
 		{ type: 'Plugin', num: thisMeta.plugins.val - lastMeta.plugins.val },
-		{ type: 'Theme', num: thisMeta.themes.val - lastMeta.themes.val },
+		{ type: 'Theme', num: thisMeta.themes?.val - lastMeta.themes?.val },
```

Comment:
Is `thisMeta.themes` the only field here that needs a null guard?

Reply:
Also, from the land of "I should've expected it already, but TIL": in JavaScript, `null - null` evaluates to `0 - 0` since null is falsy and the subtraction operator casts the operand to a Number.

Reply:
Yeah! that was the first approach that I tried it, but it was more complex than that :)
Now with this fix, that method is unused.